TerminalDocs/distributions.md

```diff
@@ -0,0 +1,103 @@
+
+## Distribution feature comparison
+
+|                                            | Packaged                 | Preinstallation Kit | Unpackaged       | Portable                      |
+| ------------------------------------------ | ------------------------ | ------------------- | ---------------- | ----------------------------- |
+| **Automatic updates**                      | ✅                       | ✅                  | ❌               | ❌                            |
+| **Automatic architecture selection**       | ✅                       | ✅                  | ❌               | ❌                            |
+| **Can be set as your default terminal**    | ✅                       | ✅                  | ❌               | ❌                            |
+| **"Open in Terminal" context menu**        | ✅                       | ✅                  | ❌               | ❌                            |
+| **Automatic start on login option**        | ✅                       | ✅                  | _manual_         | _manual_                      |
+| **Double-click installation**              | ✅                       | ❌                  | ❌               | ❌                            |
+| **Installation on non-networked machines** | ❌                       | ✅                  | ✅               | ✅                            |
+| **Preinstallation in a Windows image**     | ❌                       | ✅                  | _as plain files_ | _as plain files_              |
+| **User-controlled installation path**      | ❌                       | ❌                  | ✅               | ✅                            |
+| **Double-click activatable**               | ❌                       | ❌                  | ✅               | ✅                            |
+| **Settings storage location**              | User folder, per package | (same as packaged)  | `%LOCALAPPDATA%` | Next to `WindowsTerminal.exe` |
+
+## Windows Terminal Portable
+
+As of stable channel version 1.17, Windows Terminal supports being deployed in ["Portable mode"]. Portable mode ensures
+that all data created and maintained by Windows Terminal is saved next to the application so that it can be more easily
+moved across different environments.
+
+Portable mode is supported by the unpackaged "ZIP" distribution.
+
+This is an officially-supported mode of execution where Windows Terminal stores its settings in a `settings` folder next
+to `WindowsTerminal.exe`.
+
+Portable mode is not supported in the packaged or preinstallation kit distributions of Windows Terminal.
+
+### Why use Portable mode?
+
+The unpackaged and portable mode distributions of Windows Terminal allow you to use Terminal without installing it
+globally, e.g. on systems where you may not have permission to install MSIX packages or download software from the
+Microsoft Store.
+
+Portable mode allows you to carry around (and archive!) a preconfigured installation of Windows Terminal and run it from
```

Comment:
I think "to carry around and archive" would be better.

Reply:
I would rephase to "or archive" in that case